### PR TITLE
Adaflo coupling normal vec

### PR DESCRIPTION
**Pull request by mschreter**

### CODE REVIEW

File: include/meltpooldg/normal_vector/normal_vector_operation_adaflo_wrapper.hpp [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+          );
+        /**
+         * initialize the preconditioner
+         */
+        //initialize_mass_matrix_diagonal<dim, double>(scratch_data.get_matrix_free(),
+                                                     //scratch_data.get_constraint(
+                                                       //advec_diff_dof_idx),
+                                                     //advec_diff_dof_idx,
+                                                     //advec_diff_quad_idx,
+                                                     //preconditioner);
```
Comment (peterrum) [inline include/meltpooldg/normal_vector/normal_vector_operation_adaflo_wrapper.hpp:83]: There are a lot of commented lines in this file!?

**Review by peterrum — APPROVED** [review]

**Comment by peterrum** [comment]
CI is not happy. I think there is missing ifdef. Furthermore, could you for the time being, change the adaflo branch to `peterrum/ls_composition_2` in the GitHub action. We will revert this change later on.

**Comment by mschreter** [comment]
This PR introduces the adaflo interface to the normal_vector; The relevant parameters will be set correctly in a follow-up PR.

**Comment by peterrum** [comment]
I have checked master. Simplex is still working :)

**Pull request merged**